decimal.go

```diff
@@ -107,10 +117,18 @@ func (c *Context) NewFromString(s string) (*Decimal, Condition, error) {
 }
 
 // String is a wrapper of ToSci.
+// REVIEW: why?
```

Comment:
I just meant why have `ToSci` in the first place then. I didn't see `ToStandard` before though so feel free to ignore.